### ISSUE TITLE
Feature request: internationalization

### ISSUE THREAD

**Issue by bitwarden48684** [bitwarden/help#404]
Are there plans to internationalize this repository, either by doing a professional translation or by crowd-sourcing it from the community? I'd like to introduce friends and family to Bitwarden, but they don't understand English and the transition would be easier if I could point them to the knowledge base in the appropriate language.

**Comment by fschillingeriv** [comment]
Hey @bitwarden48684, that's a really important point that you've brought up. While we don't have a specific plan of attack yet, we're aware that this is something we definitely need to be thinking about.

I'm going to keep this issue open & provide updates as I'm able.

**Comment by vachan-maker** [comment]
I am part of the signal support centre project.
This is the link of their documentation: https://support.signal.org/hc/en-us
The documentation of Signal is pretty short and contains about 3, 000 or more strings.
The Bitwarden documentation is pretty big. Some articles have a lot of strings. Not only that the articles are updated constantly. Major changes are made within a day (which is good but kind of a pain to translate).

I am not discouraging the localization of the help centre but it will require a lot of time and effort.

The localization of the website bitwarden.com is possible. Maybe starting off with this and then going to the help centre would be a good idea.

**Comment by fschillingeriv** [comment]
Agreed @vachan-maker, Bitwarden uses [Crowdin](https://crowdin.com/project/bitwarden-web) to manage localization of apps, but that's not particularly viable for /help for all the reasons you mentioned.

**Comment by tgreer-bw** [comment]
@vachan-maker you mentioned this tool previously, perhaps this could work @fschillingeriv?

https://blog.crowdin.com/2020/12/17/website-translation-with-crowdin/?utm_source=newsletter-blog&utm_medium=email&utm_campaign=website-translation&utm_content=17_12_20
